deploy-manage/autoscaling/trained-model-autoscaling.md

```diff
@@ -2,28 +2,188 @@
+
+On Serverless, adaptive allocations are automatically enabled for all project types. However, the "Adaptive resources" control is not displayed in Kibana for Observability and Security projects.
+
+## Model deployment resource matrix [model-deployment-resource-matrix]
+
+The used resources for trained model deployments depend on three factors:
+
+* your cluster environment (Serverless, Cloud, or on-premises)
+* the use case you optimize the model deployment for (ingest or search)
+* whether model autoscaling is enabled with adaptive allocations/resources to have dynamic resources, or disabled for static resources
+
+If you use {{es}} on-premises, vCPUs level ranges are derived from the `total_ml_processors` and `max_single_ml_node_processors` values. Use the [get {{ml}} info API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-ml-info) to check these values. The following tables show you the number of allocations, threads, and vCPUs available in Cloud when adaptive resources are enabled or disabled.
+
+::::{note}
+On Serverless, adaptive allocations are automatically enabled for all project types. However, the "Adaptive resources" control is not displayed in {{kib}} for Observability and Security projects.
+::::
+
+### Deployments in Cloud optimized for ingest [_deployments_in_cloud_optimized_for_ingest]
+```{applies_to}
+stack: {{ecloud}}
+```
+
+In case of ingest-optimized deployments, we maximize the number of model allocations.
+
+#### Adaptive resources enabled [_adaptive_resources_enabled]
+
+| Level | Allocations | Threads | vCPUs |
+| --- | --- | --- | --- |
+| Low | 0 to 2 if available, dynamically | 1 | 0 to 2 if available, dynamically |
+| Medium | 1 to 32 dynamically | 1 | 1 to the smaller of 32 or the limit set in the Cloud console, dynamically |
+| High | 1 to limit set in the Cloud console *, dynamically | 1 | 1 to limit set in the Cloud console, dynamically |
+
+* The Cloud console doesn’t directly set an allocations limit; it only sets a vCPU limit. This vCPU limit indirectly determines the number of allocations, calculated as the vCPU limit divided by the number of threads.
+
+#### Adaptive resources disabled [_adaptive_resources_disabled]
+
+| Level | Allocations | Threads | vCPUs |
+| --- | --- | --- | --- |
+| Low | 2 if available, otherwise 1, statically | 1 | 2 if available |
+| Medium | the smaller of 32 or the limit set in the Cloud console, statically | 1 | 32 if available |
+| High | Maximum available set in the  Cloud console *, statically | 1 | Maximum available set in the Cloud console, statically |
+
+* The Cloud console doesn’t directly set an allocations limit; it only sets a vCPU limit. This vCPU limit indirectly determines the number of allocations, calculated as the vCPU limit divided by the number of threads.
```

Comment:
not sure this is the right way to escape an asterisk - should check it renders properly

```suggestion
\* The Cloud console doesn’t directly set an allocations limit; it only sets a vCPU limit. This vCPU limit indirectly determines the number of allocations, calculated as the vCPU limit divided by the number of threads.
```